TransformProcessor/Logic/vtkSlicerTransformProcessorLogic.cxx

```diff
@@ -441,6 +445,25 @@ void vtkSlicerTransformProcessorLogic::ComputeFullTransform( vtkMRMLTransformPro
   outputTransformNode->SetAndObserveTransformToParent( fromToToLinearTransformNode );
 }
 
+//----------------------------------------------------------------------------
+void vtkSlicerTransformProcessorLogic::ComputeInverseTransform( vtkMRMLTransformProcessorNode* paramNode )
+{
+  bool verboseWarnings = true;
+  bool conditionsMetForProcessing = this->IsTransformProcessingPossible( paramNode, verboseWarnings );
+  if ( conditionsMetForProcessing == false )
+  {
+    return;
+  }
+
+  vtkMRMLLinearTransformNode* forwardTransformNode = paramNode->GetInputForwardTransformNode();
+  // node stores the transform _to_ parent. Inverse will be the transform _from_ parent.
+  vtkSmartPointer< vtkMatrix4x4 > matrixTransformFromParent = vtkSmartPointer< vtkMatrix4x4 >::New(); 
+  forwardTransformNode->GetMatrixTransformFromParent( matrixTransformFromParent );
```

Comment:
Is there a way to invert transform between two arbitrary transform nodes? (not just between "forward" and its parent but between "forward" and any other transform)
If you don't have time right now then just add a ticket to keep in mind.

Reply:
Yes there is, see "Compute Full Transform" mode. You can set a 'from' node and a 'to' node.